pkg/services/evaluation_report/evaluation_report_deleter_test.go

```diff
@@ -0,0 +1,70 @@
+package evaluationreport
+
+import (
+	"github.com/gofrs/uuid"
+
+	"github.com/transcom/mymove/pkg/appcontext"
+	"github.com/transcom/mymove/pkg/apperror"
+	"github.com/transcom/mymove/pkg/auth"
+	"github.com/transcom/mymove/pkg/models"
+	"github.com/transcom/mymove/pkg/services"
+	"github.com/transcom/mymove/pkg/testdatagen"
+)
+
+func (suite *EvaluationReportSuite) TestEvaluationReportDeleter() {
+	setupTestData := func() (services.EvaluationReportDeleter, models.EvaluationReport, appcontext.AppContext) {
+		deleter := NewEvaluationReportDeleter()
+		move := testdatagen.MakeDefaultMove(suite.DB())
+		shipment := testdatagen.MakeMTOShipment(suite.DB(), testdatagen.Assertions{
+			MTOShipment: models.MTOShipment{
+				MoveTaskOrderID: move.ID,
+			},
```

Comment:
we can simplify this a bit and avoid creating an extra move
```suggestion
			Move: move,
```

Reply:
Fixed ✅